src/http/api/v1/exports.ts

```diff
@@ -0,0 +1,333 @@
+import { JsonStorage } from "chainsauce";
+import express from "express";
+import fs from "fs";
+
+import database from "../../../database.js";
+import { getPrices } from "../../../prices/index.js";
+import { Round } from "../../../indexer/types.js";
+import ClientError from "../clientError.js";
+
+const router = express.Router();
+
+async function exportVotesCSV(db: JsonStorage, round: Round) {
+  const csv = createObjectCsvStringifier({
+    header: [
+      "id",
+      "transaction",
+      "blockNumber",
+      "projectId",
+      "applicationId",
+      "roundId",
+      "voter",
+      "grantAddress",
+      "token",
+      "amount",
+      "amountUSD",
+      "amountRoundToken",
+    ].map((h) => ({ id: h, title: h })),
+  });
+
+  return csv
+    .getHeaderString()!
+    .concat(
+      csv.stringifyRecords(
+        await db.collection(`rounds/${round.id}/votes`).all()
+      )
+    );
+}
+
+async function exportPricesCSV(chainId: number, round: Round) {
+  const prices = await getPrices(chainId);
+
+  const pricesDuringRound = prices.filter(
+    (price) =>
+      price.timestamp >= round.roundStartTime * 1000 &&
+      price.timestamp <= round.roundEndTime * 1000
+  );
+
+  const csv = createObjectCsvStringifier({
+    header: ["token", "code", "price", "timestamp", "block"].map((h) => ({
+      id: h,
+      title: h,
+    })),
+  });
+
+  return csv.getHeaderString()!.concat(csv.stringifyRecords(pricesDuringRound));
+}
+
+type BasePassportScore = {
+  address: string;
+  score: string | null;
+  status: string;
+  last_score_timestamp: string;
+};
+
+type FullPassportScore = BasePassportScore & {
+  evidence: {
+    type: string;
+    success: boolean;
+    rawScore: string;
+    threshold: string;
+  } | null;
+  error?: string;
+};
+
+type FlatPassportScoreWithCoefficient = BasePassportScore & {
+  coefficient: 0 | 1;
+  type?: string;
+  success?: boolean;
+  rawScore?: string;
+  threshold?: string;
+};
+
+type PassportScoresMap = {
+  [address: string]: FlatPassportScoreWithCoefficient;
+};
+
+async function exportVoteCoefficientsCSV(db: JsonStorage, round: Round) {
+  const processPassportScores = (
+    scores: FullPassportScore[]
+  ): PassportScoresMap => {
+    return scores.reduce((map, score) => {
+      const address = score.address.toLowerCase();
+      const { evidence, error, ...remainingScore } = score;
+      const coefficient = evidence !== null && evidence.success ? 1 : 0;
+      map[address] = { ...remainingScore, ...evidence, coefficient };
+      return map;
+    }, {} as PassportScoresMap);
+  };
+
+  const [votes, data] = await Promise.all([
+    db.collection(`rounds/${round.id}/votes`).all(),
+    fs.promises.readFile("./data/passport_scores.json", {
+      encoding: "utf8",
+      flag: "r",
+    }),
+  ]);
+
+  const passportScores: FullPassportScore[] = JSON.parse(data);
+  const passportScoresMap = processPassportScores(passportScores);
+
+  const records = votes.map((vote: any) => {
```

Comment:
Are we still unsure of the shape? `Vote` type is pretty well established at this point, I think. 

```typescript
type Vote = {
  id: string;
  projectId: string;
  applicationId: string;
  roundId: string;
  token: string;
  voter: string;
  grantAddress: string;
  amount: string;
  amountUSD: string;
};
```